src/device-registry/models/Device.js

```diff
@@ -136,6 +136,12 @@ const deviceSchema = new mongoose.Schema(
       trim: true,
       unique: true,
     },
```

Comment:
addition of the category field to the Device schema. Used to categorise devices.